### ISSUE TITLE
Could your provide training script

### ISSUE THREAD

**Issue by tonysy** [raghudeep/netwarp_public#1]
Hi, thanks for your work, I would like to know could you provide the training code, thx.

**Comment by chowkamlee81** [comment]
Can u kindle provide scripts to do training which will be helpful. Kindly do the needful

**Comment by lxtGH** [comment]
Thanks for your work, could you provide the training code? much thanks

**Comment by lxtGH** [comment]
I think the model that he provided isn't consistent with his paper

**Comment by burhanmudassar** [comment]
@lxtGH I agree. What metric results do you get on cityscapes validation set?
